framework/src/mesh/MooseMesh.C

```diff
@@ -2048,6 +2048,7 @@ MooseMesh::buildMeshBaseObject(ParallelType override_type)
 
     mesh = libmesh_make_unique<ReplicatedMesh>(_communicator, dim);
   }
+  mesh->remove_ghosting_functor(mesh->default_ghosting());
```

Comment:
@permcody @friedmud I guess that #13145 really didn't remove default ghosting haha. Looks like there's only [24 tests failing](https://civet.inl.gov/job/375328/) though, so we should be able to truly remove default ghosting pretty soon!